frontend/src/bundles/common/components/checkbox/checkbox.tsx

```diff
@@ -47,6 +47,7 @@ const Checkbox = <T extends FieldValues>({
                         name={name}
                         type="checkbox"
                         id="toggle-checkbox"
+                        aria-label="Toggle Checkbox"
```

Comment:
I don't think thats very valuable description. Could we pass as props screen-reader only label for checkboxes?